Source/nuspecs/Prism.Ninject.nuspec

```diff
@@ -22,10 +22,33 @@
         <dependency id="CommonServiceLocator" version="1.3.0" />
         <dependency id="Prism.Wpf" version="$wpfVersion$" />
       </group>
+
+      <group targetFramework="WindowsPhone8.0">
+        <dependency id="Prism.Forms" version="$formsVersion$" />
+        <dependency id="CommonServiceLocator" version="1.3" />
+        <dependency id="Xamarin.Forms" version="1.5.0.6447" />
+        <dependency id="Portable.Ninject" version="3.3.1" />
```

Comment:
The nuspec is cribbed from the Unity version. It adds a dependency on Portable.Ninject which was ported by Oren Novotny.